plugin/src/integrationTest/kotlin/com/vanniktech/maven/publish/MavenPublishPluginIntegrationTest.kt

```diff
@@ -248,6 +248,43 @@ class MavenPublishPluginIntegrationTest {
     assertPomContentMatches(linuxArtifactId)
   }
 
+  @Test fun generatesArtifactsAndDocumentationOnKotlinMppWithAndroidDokkaProject() {
+    setupFixture("passing_kotlin_mpp_with_android_dokka_project")
+
+    val result = executeGradleCommands(TEST_TASK, "--stacktrace", "--stacktrace")
+
+    assertExpectedTasksRanSuccessfully(result, hasDokka = true)
+
+    assertExpectedCommonArtifactsGenerated()
+    assertPomContentMatches()
+
+    repoFolder.walkTopDown().forEach { println(it.path) }
```

Comment:
Debugging purposes or should we keep it?

Reply:
That was for debugging